articles/container.html

```diff
@@ -1491,19 +1491,24 @@ <h3>Convert Slice to Array Pointer</h3>
 type P *A
 
 func main() {
-    var x = make([]int, 3, 5)
-    var _ = (*[3]int)(x) // okay
-    var _ = (*[2]int)(x) // okay
-    var _ = (*A)(x)      // okay
-    var _ = P(x)         // okay
+    var x []int
+    var y = make([]int, 0)
+    var x0 = (*[0]int)(x) // okday, x0 == nil
+    var y0 = (*[0]int)(y) // okay, y0 != nil
     
-    var y = S(x)
-    var _ = (*[3]int)(y) // okay
-    var _ = (*[2]int)(y) // okay
-    var _ = (*A)(y)      // okay
-    var _ = P(y)         // okay
+    var z = make([]int, 3, 5)
+    var _ = (*[3]int)(z) // okay
+    var _ = (*[2]int)(z) // okay
+    var _ = (*A)(z)      // okay
+    var _ = P(z)         // okay
     
-    var _ = (*[4]int)(x) // will panic
+    var t = S(x)

```

Comment:
S(z)? Othewise, following lines will panic.

BTW, how about to use w instead of t?

Reply:
Yeah, `w` is better. Done.